src/Aspire.Dashboard/README.md

```diff
@@ -79,6 +81,8 @@ For API key authentication, set `Dashboard:Otlp:AuthMode` to `ApiKey`, then add
 - `Dashboard:Otlp:PrimaryApiKey` specifies the primary API key. (required, string)
 - `Dashboard:Otlp:SecondaryApiKey` specifies the secondary API key. (optional, string)
 
+For more information, see [Security considerations for running the .NET Aspire dashboard: Secure telemetry endpoint](https://learn.microsoft.com/dotnet/aspire/fundamentals/dashboard/security-considerations#secure-telemetry-endpoint).
```

Comment:
PrimaryApiKey/SEcondaryApiKey are at https://learn.microsoft.com/en-us/dotnet/aspire/fundamentals/dashboard/configuration#otlp-authentication of course. Will they see that higher up the doc?